torch/_tensor_docs.py

```diff
@@ -3149,36 +3155,39 @@ def callable(a, b) -> number
 
 Args:
     dim (int): the axis along which to index
-    index (LongTensor): the indices of elements to scatter,
-      can be either empty or the same size of src.
-      When empty, the operation returns identity
-    src (Tensor): the source element(s) to scatter,
-      incase `value` is not specified
-    value (float): the source element(s) to scatter,
-      incase `src` is not specified
-    reduce (string): reduction operation to apply,
-      can be either 'add' or 'multiply'.
+    index (LongTensor): the indices of elements to scatter, can be either empty
+        or the same size of ``src``. When empty, the operation returns
```

Comment:
This contradicts the statement above that `index.size(d) <= src.size(d)`

Reply:
Good catch. I'll change "same size of" to "same dimensionality as".

The example has `index.size(d) < src.size(d)`, that works. I verified empty tensor works as well.

Reply:
And the `scatter_add_` docstring contains the same issue, I'll fix it there as well.